websites/W/WebtoonXYZ/presence.ts

```diff
@@ -0,0 +1,84 @@
+const presence = new Presence({
+  clientId: "836962986451140609"
+}),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const v2icons = await presence.getSetting("v2icons"),
+        logo = await presence.getSetting("logo"),
+        buttons = await presence.getSetting("buttons"),
+        data: PresenceData = {
+          largeImageKey: (logo) == 0 ? "logo" : "logo-v2",
```

Comment:
```suggestion
          largeImageKey: !logo ? "logo" : "logo-v2",
```
but i don't really see sense in two types of images :/